Gui/opensim/view/src/org/opensim/view/nodes/OneMarkerNode.java

```diff
@@ -199,6 +199,7 @@ public String getUndoPresentationName() {
        ExplorerTopComponent.addUndoableEdit(auEdit);
 
         obj.changeFramePreserveLocation(context.getCurrentStateRef(), PhysicalFrame.safeDownCast(comp));
+        obj.getSocket("parent_frame").finalizeConnection(obj.getRoot());
```

Comment:
Can we move this inside the changeFramePreserveLocation method to avoid expicit finalizeConnection here? Also do we need the same for PathPoint changes?

Reply:
I know it's not clearly documented but I think `finalizeConnection()` should be called by API users. There isn't anywhere that the API calls `Component::finalizeConnections()` or `AbstractSocket::finalizeConnection()` internally (except `Model::initSystem()`).

As for PathPoints, I think I looked into this and it wasn't necessary.